text-show-instances.cabal

```
@@ -156,6 +159,7 @@ library
                      , text-show             >= 3.4    && < 4
                      , time                  >= 0.1    && < 1.12
                      , unordered-containers  >= 0.2    && < 0.3
+                     , uuid-types            >= 1      && < 2
```

I'd prefer slightly to use the secondary version number for the upper version bound:

```suggestion
                     , uuid-types            >= 1      && < 1.1
```